types/rusha/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for rusha 0.8
+// Project: https://github.com/srijs/rusha#readme
+// Definitions by: Jacopo Scazzosi <https://github.com/jacoscaz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+interface Hash {
+    update(value: string|number[]|ArrayBuffer|Buffer): Hash;
+    digest(encoding?: undefined): ArrayBuffer;
+    digest(encoding: 'hex'): string;
+}
+
+interface WorkerRequest {
+    id: string;
+    data: string|number[]|ArrayBuffer|Buffer;
+}
+
+interface WorkerResponse {
+    id: string;
+    hash: string;
+}
+
+interface Worker {
```

Comment:
Wondering if this could/should inherit from the built-in `Worker` type?

Reply:
Sounds good. I’ll make the change.

Reply:
Done!